modules/get-started/pages/whats-new.adoc

```diff
@@ -89,6 +93,7 @@ The following commands have improved documentation or examples:
 
 == Documentation enhancements
 
+- A new xref:reference:glossary.adoc[glossary] provides definitions helpful for working with Redpanda. In text, terms with hover text and a link to the glossary entry render with a dotted line; for example, glossterm:controller[].
```

Comment:
Nit: Instead of "in text," would something like "on each page / doc" also work? "In text" sounds a bit ambiguous to me

Reply:
yes, agreed. I just removed it, so now: "Terms with hover text and ..."